CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 -   Fix windows build.
--   Return an error when creating a halight swap if lnd certificate or macaroon are unavailable instead of failing silently.
+-   Return an error when creating a halbit swap if lnd certificate or macaroon are unavailable instead of failing silently.
 -   Only return a single action on the get swaps endpoint. This is now the recommended action available to the user.
```

Comment:
This one is not "fixed" compared to the previous version (0.7.3) because the endpoint this is addressing doesn't even exist in the previous version. For end users, this is the normal behaviour of the newly introduced endpoint.

Not sure if anyone actually reads our changelogs but that is something to keep in mind when writing them.

cc @D4nte

Reply:
If we want to mention this somewhere, then I would suggest a release announcement blog post where we have more real estate to talk about this.

Reply:
Cool, will remove. Was not added by this patch but its worth cleaning up. Will do it as patch 0 on this PR and force push.